bot/utils/blurple_formatter.py

```diff
@@ -278,9 +347,9 @@ def unparse(node: ast.AST, nl_able: bool = False) -> str:
         nl_after = (op_type not in nl_before_op) * "\n"
 
         return (
-            f"({(parenthesize if par_left else unparse)(left, True)}{space()}"
+            f"({space()}{(parenthesize if par_left else unparse)(left, True)}{space()}"
```

Comment:
Didn't you also add `space()` inside `parenthesize`?

Reply:
Both spaces matter. Example, when `((1 + 2) * 3)` is done.